docs/DafnyRef/UserGuide.md

```diff
@@ -220,13 +215,13 @@ default, but can be skipped by using the `-noVerify` or `-dafnyVerify:0` option
 successful but can be skipped or always executed by using variations of the `-compile` option.
 - Whether the source code of the compiled target is written out is controlled by `-spillTargetCode`
 - The particular target language used is chosen by `-compileTarget`
-- Whether or not the dafny tool attempts to run the compiled code is controlled by `-compile`
+- Whether or not the `dafny` tool attempts to run the compiled code is controlled by `-compile`
 
 The dafny tool terminates with these exit codes:
 
 * 0 -- success
 * 1 -- invalid command-line arguments
-* 2 -- parse or types errors
+* 2 -- syntax, parse or type errors
```

Comment:
What's the difference between a syntax and a parse error? I think we can drop `parse` from this list.

Reply:
Okay, but from the user's perspective there's not first a lexing and then a parsing phase, there's just a step that determines if the input program matches the Dafny grammar, right?